gef.py

```diff
@@ -297,8 +302,32 @@ def reset_all_caches():
     return
 
 
+def highlight_text(x):
+    if (not len(HIGHLIGHT_TABLE)):
+        return x
+
+    if HIGHLIGHT_REGEX_ENABLED:
+        for repl, color in HIGHLIGHT_TABLE.items():
```

Comment:
why `repl`? that's a common word in programming that doesn't make sense in this context.